src/swarmClient/main.js

```diff
@@ -39,16 +36,6 @@ module.exports = {
         onclose = (onclose && once(onclose)) || (() => {});
 
 
-
-        if(public_pem) {
-            // throws an error if key is malformed
```

Comment:
We're losing this up-front verification, correct? Seems like a regression in startup behaviour.

Reply:
It was resolved to load and verify keys at startup. See main.js:34 and main.test.js:376-411.

Reply:
It was unclear that the conversation mentioned was in relation to this issue, my apologies. I stand by my original statement that we should not be regressing startup behaviour, especially in an unrelated commit/ticket/topic. This PR is only to address crypto efficiencies, and this is not a necessary change for that.

If we continue with the verification approach, we also need to change documentation and suggestions that, for example, you don't need a private key (or valid key pair at all) to perform non-write operations. Let's properly story this out and qualify it before implementing it.

Reply:
> need to change documentation 

The existing documentation is consistent with this change. 

> ...suggestions that, for example, you don't need a private key (or valid key pair at all) to perform non-write operations

The agreed-upon method was for a valid key pair to be necessary to create a client object. 

> unrelated commit/ticket/topic

All crypto behavior is relevant to this ticket.

The startup needs to be re-implemented with the new crypto; that's why I needed to clarify the expected behavior.